crates/biome_js_analyze/tests/specs/nursery/useExplicitType/invalid.ts.snap

```diff
@@ -184,25 +193,6 @@ invalid.ts:5:1 lint/nursery/useExplicitType ━━━━━━━━━━━━
   i Add a return type to the function.
   
 
-```
-
-```
-invalid.ts:9:7 lint/nursery/useExplicitType ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
-
-  × The variable doesn't have a type defined.
-  
-     7 │ }
-     8 │ 
-   > 9 │ const fn = function () {
-       │       ^^
-    10 │ 	return 1;
-    11 │ };
```

Comment:
Is this a regression or not? 
- If it's a regression, we should fix it
- If it's not a regression, we need to remove this from the original code 

This comment is valid for all the deletions that belong to this snapshot.

Reply:
Sorry for the long delay, I'm now updating the snapshots, but this file is already fine. The deletion you observe correspond to **duplicated** diagnostics - scroll further to see another error on line 9. Initial rule implementation flagged both the variable assignment and the untyped function, which is obviously redundant. So this is still a piece of invalid code we flag, but with one error message instead of two.